### ISSUE TITLE
Public - FE - Terms page

### ISSUE THREAD

**Issue by ludtkemorgan** [bloom-housing/bloom#4421]
Setup the page and pull over the content from Detroit's terms page https://homeconnect.detroitmi.gov/terms.

Steps needed:
- Create a new page within the new [Detroit repo](https://github.com/bloom-housing/bloom-detroit) for the terms page
- Copy the content from [Detroit code base](https://github.com/CityOfDetroit/bloom/blob/dev/sites/public/src/md_content/terms.md)

**Comment by sarahlazarich** [comment]
@ludtkemorgan can we assign matthew to this one? Not sure if he has access yet

**Comment by ludtkemorgan** [comment]
@sarahlazarich Yep! I'll add more description here and make sure they have access

**Comment by sarahlazarich** [comment]
@matzduniuk or @ludtkemorgan can you please leave QA notes on this?

**Comment by sarahlazarich** [comment]
I can't find the place I should be looking for this on the new Detroit site

**Comment by ludtkemorgan** [comment]
@sarahlazarich oops, forgot to add the QA notes after it got deployed

This can be tested on the new Detroit site: https://bloom-detroit.netlify.app/terms

Notes: 
- There aren't any links to this page yet so you can only get to it with direct link
- Detroit does not have this page in any language besides English so same behavior is carried over to this site

**Comment by sarahlazarich** [comment]
This matches, looks good!
